tests/multicard/test_offline_inference_distributed.py

```diff
@@ -61,3 +61,22 @@ def test_models_distributed_DeepSeek():
             distributed_executor_backend="mp",
     ) as vllm_model:
         vllm_model.generate_greedy(example_prompts, max_tokens)
+
+
+def test_deepseek_model_with_dbo():
```

Comment:
```suggestion
@patch.dict(os.environ, {"VLLM_ASCEND_ENABLE_DBO": "1"})
def test_deepseek_model_with_dbo():
```

we need restore the env after test excuted.

[1] https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch.dict